scripts/cluster.py

```diff
@@ -244,7 +244,7 @@ def _update_user_code(self, user_source_directory):
       raise Exception("Directory {} does not exist.".format(user_source_directory))
     # If user_source_directory is "/a/b/c", then local_directory_name is "c".
     local_directory_name = os.path.split(os.path.realpath(user_source_directory))[1]
-    remote_directory = os.path.join(self.installation_directory, "user_source_files", local_directory_name)
+    remote_directory = os.path.join("user_source_files", local_directory_name)
```

Comment:
What was it doing wrong before? This should work out to something like `"$HOME/user_source_files/ray_example_code`, right?